docs/.vuepress/config.ts

```diff
@@ -10,17 +10,6 @@ export default defineUserConfig({
   description: "MAA",
   port: 3001,
 
-  locales: {
-    "/": {
-      lang: "zh-CN",
-      description: "开发者文档",
-    },
-    "/en-us/": {
-      lang: "en-US",
-      title: "Documents",
-    },
-  },
-
```

Comment:
This completely broke the language bar. Need to revert (plus we can finally switch languages without going back to home screen)

Reply:
it's because the english title is different from the others.

u mean, if i change language in subpage, it will return to the home page?

![Screenshot_2024-05-13-16-58-31-944_com.android.chrome-edit.jpg](https://github.com/MaaAssistantArknights/MaaAssistantArknights/assets/55343783/757590e2-21d6-4b9d-87d0-cb029a6d7b3a)

![Screenshot_2024-05-13-16-58-38-885_com.android.chrome-edit.jpg](https://github.com/MaaAssistantArknights/MaaAssistantArknights/assets/55343783/c058f799-03f9-4d30-bfdb-0e1df5c8d0bb)

Reply:
That was the case, but not anymore. Honestly the best solution (like big sites) even the default language has it's own lang-LANG, that way it's easy to switch languages.
The only remaining things to make this work is changing the links to docs/zh-cn, instead of just docs (maybe we can put a generic default that if it's only docs/ it goes to docs/zh-cn automatically